### PR TITLE
initial commit, changed banner

### PR DESCRIPTION
**Pull request by khxia**
#103 Actually, not sure if changing to banner in the Homepage falls under #103.

### CODE REVIEW

**Review by jodymlin — COMMENTED** [review]
**A few SVG things**:
image
* some weird shadows under that get chopped off and look funny. 2 things we can do here, either make the svg height reach all the way to the bottom of the banner, or (need ur advice @GalenWong) remove the shadows.


image
* the google home/alexa i think is missing part of the wiring/plug. Perhaps got deleted when trying to export it. The top of the wiring also doesn't go to the top of the banner and gets cutoff early. 


* The SVG on the right also seems a little bigger than the one on the left. I noticed because the code blocks on the right look bigger the code blocks on the left. Not sure if it just turned out this way when the SVGs were exported. If it's too hard to fix, we can always try fix it later. 


**Stuff with Banner cutoff**:
Earlier we discussed ways to fix the awkward code block cutoffs on the. I have an idea (@GalenWong need your design wisdom): What if we edited the SVG code blocks in the background so that they all end before getting cutoff. Then we re-export the SVG and use that. 

**ACM Hack Wordmark**
Design recreated the wordmarks so that there shouldn't be issues rendering them. The working versions should be in the ACM at UCLA drive. We can probably use this wordmark to fix the banner.

**Comment by khxia** [comment]
> **A few SVG things**:
> image
> 
> * some weird shadows under that get chopped off and look funny. 2 things we can do here, either make the svg height reach all the way to the bottom of the banner, or (need ur advice @GalenWong) remove the shadows.
> 
> image
> 
> * the google home/alexa i think is missing part of the wiring/plug. Perhaps got deleted when trying to export it. The top of the wiring also doesn't go to the top of the banner and gets cutoff early.
> * The SVG on the right also seems a little bigger than the one on the left. I noticed because the code blocks on the right look bigger the code blocks on the left. Not sure if it just turned out this way when the SVGs were exported. If it's too hard to fix, we can always try fix it later.
> 
> **Stuff with Banner cutoff**:
> Earlier we discussed ways to fix the awkward code block cutoffs on the. I have an idea (@GalenWong need your design wisdom): What if we edited the SVG code blocks in the background so that they all end before getting cutoff. Then we re-export the SVG and use that.
> 
> **ACM Hack Wordmark**
> Design recreated the wordmarks so that there shouldn't be issues rendering them. The working versions should be in the ACM at UCLA drive. We can probably use this wordmark to fix the banner.

**Comment by khxia** [comment]
Fixed most of the issues. 

Except:
- For me, the ACM Hack Wordmark still doesn't work even with the new svg on the UCLA at ACM drive. Now, instead of showing a pink block, the "hack" doesn't even render at all
- For the banner cutoff idea, I already did what @jodymlin suggested regarding the cutoffs for code blocks on the top and bottom. (I think that code block cutoffs horizontally are part of the design, unless I'm wrong)

**Comment by GalenWong** [comment]
> * For me, the ACM Hack Wordmark still doesn't work even with the new svg on the UCLA at ACM drive. Now, instead of showing a pink block, the "hack" doesn't even render at all

I will take a look. It might take some rework of the logo.

**Comment by GalenWong** [comment]
@khxia i uploaded another logo that should work in the browser in the drive. try again.

**Comment by GalenWong** [comment]
> For the banner cutoff idea, I already did what @jodymlin suggested regarding the cutoffs for code blocks on the top and bottom. (I think that code block cutoffs horizontally are part of the design, unless I'm wrong)

We don't want the horizontal cut of (at least I don't). We can just let the long code blocks overflow out horizontally. And I actually don't mind the SVG for the IDE to overflow out a bit as well. Let me know if you need SVG assets.

File: src/components/HomePage/Banner.js [hunk 1]
```diff
@@ -4,104 +4,97 @@ import { withStyles } from '@material-ui/core/styles';
...
 
-function Wrenches({ classes }) {
-	return <div className={classes.wrenches}>
-		<SvgImg className={classes.bigWrench} src={bigWrenches} width={936} height={724} />
-		<SvgImg className={classes.smallWrench} src={smallWrenches} width={113} height={216} />
-	</div>;
+function LeftBanner({ classes }) {
+	return (
+		<SvgImg imgStyle={{ width: 'auto' }} className={classes.rightBanner} src={leftbanner}
+			width={500} height={800} />
```
Comment (GalenWong) [inline src/components/HomePage/Banner.js:58]:
```suggestion
		<SvgImg
		  imgStyle={{ width: 'auto' }}
		  className={classes.rightBanner}
		  src={leftbanner}
			width={500}
			height={800}
		/>
```

File: src/components/HomePage/Banner.js [hunk 2]
```diff
@@ -4,104 +4,97 @@ import { withStyles } from '@material-ui/core/styles';
...
+}
+
+LeftBanner.propTypes = {
+	classes: PropTypes.object.isRequired
+};
+
+function RightBanner({ classes }) {
+	return (
+		<SvgImg imgStyle={{ width: 'auto' }} className={classes.leftBanner} src={rightbanner}
+			width={500} height={800} />
```
Comment (GalenWong) [inline src/components/HomePage/Banner.js:69]:
```suggestion
		<SvgImg 
		  imgStyle={{ width: 'auto' }}
		  className={classes.leftBanner}
		  src={rightbanner}
			width={500}
			height={800}
		/>
```

File: src/components/HomePage/Banner.js [hunk 3]
```diff
@@ -4,104 +4,97 @@ import { withStyles } from '@material-ui/core/styles';
...
+	);
+}
+
+RightBanner.propTypes = {
+	classes: PropTypes.object.isRequired
+};
+
+function Wordmark({ classes }) {
+	return (
+		<SvgImg className={classes.wordmark} src={wordmark} width={1300} height={272.9} />
```
Comment (GalenWong) [inline src/components/HomePage/Banner.js:79]: add a comment here to explain why the hardcoded width and height

**Review by GalenWong — CHANGES_REQUESTED** [review]

**Review by GalenWong — APPROVED** [review]

**Pull request merged**